scripts/build.sh

```diff
@@ -142,8 +153,8 @@ if [[ -f "$ID_PRIV_KEY" ]]; then
 elif [[ -f "$ID_PUB_KEY" ]]; then
   echo "WARNING: ID Public Key already exists for Broker Daemon. Skipping ID generation"
 elif [ "$NO_IDENTITY" != "true" ]; then
-  openssl ecparam -name prime256v1 -genkey -noout -out $ID_PRIV_KEY
-  openssl ec -in $ID_PRIV_KEY -pubout -out $ID_PUB_KEY
+  openssl ecparam -name prime256v1 -genkey -noout > $ID_PRIV_KEY
```

Comment:
does `-noout` affect this? at first glance it looks weird

Reply:
`do not print the ec parameter`, we want the key only.